packages/cli/src/utils/miscellaneous.ts

```diff
@@ -536,7 +538,8 @@ export async function sendTelemetry(
   has_config: boolean | undefined,
   spec_version: string | undefined,
   spec_keyword: string | undefined,
-  spec_full_version: string | undefined
+  spec_full_version: string | undefined,
+  respect_x_security_auth_types: string[] | undefined
```

Comment:
It looks like it'd be better to arrange all information regarding spec into a single object, what do you think?